website/docs/user_attributes.md

```diff
@@ -0,0 +1,166 @@
+|        **multiplicity** | No                                                                                              |
+|        **availability** | Always                                                                                          |
+|             **example** | _4a58cb0338fb68060ba75e20e02a4844148f9c63a5cd9a7418dc5a9a6ab641cbaef73b4c65519286359f766c45bdbad36cc9189e75a03bd12f13526c34cab93e@eid-proxy.aai.grnet.gr_   |
+|               **notes** | Use **EGI ID** within your application as the unique-identifier key for the user                |
+|              **status** | Stable                                                                                          |
+
+<!-- markdownlint-enable line-length no-inline-html -->
+
+### 2. Person Identifier
+
+|          attribute name | Person Identifier                                                                   |
+| ----------------------: | :---------------------------------------------------------------------------------- |
+|         **description** | The user's person identifier                                                        |
+|   **SAML Attribute(s)** | `http://eidas.europa.eu/attributes/legalperson/PersonIdentifier` (PersonIdentifier) |
+|          **OIDC scope** | `profile`                                                                           |
+|       **OIDC claim(s)** | `person_identifier`                                                                 |
+| **OIDC claim location** | Userinfo endpoint                                                                   |
+|              **origin** | Provided by user's Identity Provider                                                |
+|             **changes** | Yes                                                                                 |
+|        **multiplicity** | Single-valued                                                                       |
+|        **availability** | Always                                                                              |
+|             **example** | _CA/CA/UniqueId1_                                                                   |
+|               **notes** | -                                                                                   |
+|              **status** | Stable                                                                              |
+
+### 3. Given Name
+
+|          attribute name | Given Name                           |
+| ----------------------: | :----------------------------------- |
+|         **description** | The user's first name                |
+|   **SAML Attribute(s)** | `urn:oid:2.5.4.42` (givenName)       |
+|          **OIDC scope** | `profile`                            |
+|       **OIDC claim(s)** | `given_name`                         |
+| **OIDC claim location** | Userinfo endpoint                    |
+|              **origin** | Provided by user's Identity Provider |
```

Comment:
Question: Is this mapped from the eIDAS FirstName attribute in the case of natural person authentication through eIDAS?